mobile-academy/src/main/java/org/motechproject/nms/mobileacademy/service/impl/MobileAcademyServiceImpl.java

```diff
@@ -28,6 +36,10 @@
 @Service("mobileAcademyService")
 public class MobileAcademyServiceImpl implements MobileAcademyService {
 
+    private static final String COURSE_CONTENT_FILE = "nmsCourse.json";
+
+    private static final String COURSE_NAME = "MobileAcademyCourse";
+
     private static final String FINAL_BOOKMARK = "Chapter11_Quiz";
```

Comment:
Is there no spec requirement for these values to be configurable? It would be unfortunate if the course was changed to have 12 chapters and that required a recompile due to hard-coded constants...